queries.sql

```diff
@@ -51,3 +51,46 @@ SELECT species.name, COUNT(species.name) AS number_of_animals FROM animals JOIN
 SELECT full_name, animals.name AS animal_name, species.name AS type From animals JOIN owners ON animals.owner_id = owners.id JOIN species ON animals.species_id = species.id WHERE (owners.full_name IN ('Jennifer Orwell') AND species.name IN ('Digimon'));
 SELECT full_name, animals.name AS animal_name, escape_attempts From animals JOIN owners ON animals.owner_id = owners.id WHERE (full_name IN ('Dean Winchester') AND  escape_attempts = 0);
 SELECT full_name, COUNT(owners.full_name) as highest_animal_count FROM animals INNER JOIN owners ON animals.owner_id = owners.id GROUP BY full_name ORDER BY highest_animal_count DESC LIMIT 1;
+
+
+CREATE TABLE vets (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(250) NOT NULL,
+    age INT NOT NULL,
+    date_of_graduation DATE NULL,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE vets (id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(250) NOT NULL,
+    age INT NOT NULL,
+    date_of_graduation DATE NULL,
+    PRIMARY KEY(id)
+);
+
+-- Create a  joint table for vets and species
+CREATE TABLE specializations (
+    vet_id INT REFERENCES vets(id),
+    specie_id INT REFERENCES species(id),
+    PRIMARY KEY (vet_id, specie_id)
+);
+
+CREATE TABLE visits (
+    animal_id INT REFERENCES animals(id),
+    vet_id INT REFERENCES vets(id),
+    visit_date DATE,
+    PRIMARY KEY (animal_id, vet_id, visit_date)
+);
```

Comment:
- [ ] I noticed that these commands to create tables have already been included in the schema.sql file. Please kindly remove it from this file.

Reply:
well resolved! ✔️